src/runtime/virtcontainers/qemu_s390x.go

```diff
@@ -83,7 +83,11 @@ func newQemuArch(config HypervisorConfig) (qemuArch, error) {
 	}
 
 	if config.ImagePath != "" {
-		q.kernelParams = append(q.kernelParams, commonVirtioblkKernelRootParams...)
+		kernelParams, err := GetKernelRootParams(config.RootfsType, false, false)
```

Comment:
here should be 
`kernelParams, err := GetKernelRootParams(config.RootfsType, true, false)` ?

Reply:
It was this problem that caused the failure of the s390x job. I have modified it correctly.